admin.php

```diff
@@ -59,8 +59,10 @@
 	}
 	$result = mysqli_query($connection,
 		'CREATE TABLE IF NOT EXISTS `'.$settings['db_prefix'].'torrents` (' .
-			'`name` varchar(255) NOT NULL,' .
-			'`info_hash` varchar(40) NOT NULL' .
+			//'`name` varchar(255) NOT NULL,' .			// it seems useless
```

Comment:
I think we should drop the `NOT NULL` since it _is_ optional as far as the rest of the program is concerned. It is just for a nice name value when integrating elsewhere.